frontend/src/i18n/locales/de.yaml

```diff
@@ -293,11 +293,14 @@ manage:
       actions:
         title: Berechtigung
         read: Lesen
-        read-description: Nutzer mit Lesezugriff können das Video anschauen.
-        write: Schreiben
+        read-description_one: Nutzer kann das Video anschauen und Metadaten sehen.
+        read-description_other: Nutzer können das Video anschauen und Metadaten sehen.
+        write: Lesen & schreiben
         large-group-warning: Sie geben einer großen Gruppe Schreibzugriff. Ist das beabsichtigt?
-        write-description: >
-          Nutzer mit Schreibzugriff können das Video anschauen und bestimmte Videodetails editieren.
+        write-description_one: >
+          Nutzer kann das Video editieren und Metdataten sowie andere Videodetails verändern.
+        write-description_other: >
+          Nutzer können das Video editieren und Metdataten sowie andere Videodetails verändern.
```

Comment:
What happened here with "Metadaten"? :D

Also, the English "also" is missing here. Maybe "Nutzer kann außerdem..."?